src/modules/launcher/Plugins/Community.PowerToys.Run.Plugin.VSCodeWorkspaces/Main.cs

```diff
@@ -174,6 +175,105 @@ public void Init(PluginInitContext context)
             _context = context;
         }
 
+        public List<ContextMenuResult> LoadContextMenus(Result selectedResult)
+        {
+            if (selectedResult?.ContextData is not VSCodeWorkspace workspace)
+            {
+                return new List<ContextMenuResult>();
+            }
+
+            string realPath = SystemPath.RealPath(workspace.RelativePath);
+
+            return new List<ContextMenuResult>
+            {
+                CreateContextMenuResult(
+                    title: $"{Resources.CopyPath} (Ctrl+C)",
+                    glyph: "\xE8C8", // Copy
+                    acceleratorKey: Key.C,
+                    acceleratorModifiers: ModifierKeys.Control,
+                    action: () => CopyToClipboard(realPath)),
+                CreateContextMenuResult(
+                    title: $"{Resources.OpenInExplorer} (Ctrl+Shift+F)",
+                    glyph: "\xEC50", // File Explorer
+                    acceleratorKey: Key.F,
+                    acceleratorModifiers: ModifierKeys.Control | ModifierKeys.Shift,
+                    action: () => OpenInExplorer(realPath)),
+                CreateContextMenuResult(
+                    title: $"{Resources.OpenInConsole} (Ctrl+Shift+C)",
+                    glyph: "\xE756", // Command Prompt
+                    acceleratorKey: Key.C,
+                    acceleratorModifiers: ModifierKeys.Control | ModifierKeys.Shift,
+                    action: () => OpenInConsole(realPath)),
+            };
+        }
+
+        private ContextMenuResult CreateContextMenuResult(string title, string glyph, Key acceleratorKey, ModifierKeys acceleratorModifiers, Func<bool> action)
```

Comment:
I didn't quite understand.

Are you suggesting removing the `CreateContextMenuResult()` method and directly adding the entries into the `List<ContextMenuResult>()` instead? Similar to [this example](https://github.com/microsoft/PowerToys/blob/fbd72cc1eababcb5a8db54f83bf84528a591a899/src/modules/launcher/Plugins/Microsoft.Plugin.Program/Programs/UWPApplication.cs#L163C1-L205C1)?

Reply:
Yes. Otherwise we have duplicated code that acts only as transport code. This is not really needed.

Alternatively you can move all the context menu code into a method and only call the method from the property.